jobs/build/update-3.x-base-images/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+#➜  ~ docker run -it brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/jboss/openjdk18-rhel7:latest id
+#uid=185(jboss) gid=185(jboss) groups=185(jboss)
+#➜  ~ docker run -it  brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/elasticsearch:latest id
+#uid=1000(elasticsearch) gid=1000(elasticsearch) groups=1000(elasticsearch)
+#➜  ~  docker run -it brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/ansible-runner:latest  id
+#uid=0(root) gid=0(root) groups=0(root)
+# `rhel7` is root 0`
+# `nodejs6` is 1001`
+
+build_common() {
+    TAG="rhaos-4.0-rhel-7"
+    URL="http://pkgs.devel.redhat.com/cgit/containers/openshift-enterprise-base/plain/.oit/signed.repo?h="
+    USER_USERNAME="--user=ocp-build"
+    REPO=${URL}+${TAG}
```

Comment:
results in ` 'yum_repourls': ['http://pkgs.devel.redhat.com/cgit/containers/openshift-enterprise-base/plain/.oit/signed.repo?h=+rhaos-4.0-rhel-7']`
```suggestion
    REPO="${URL}${TAG}"
```
or since TAG and URL aren't going to change, just use one var :)